improver_tests/nbhood/recursive_filter/test_RecursiveFilter.py

```diff
@@ -46,6 +46,16 @@
 from ..nbhood.test_BaseNeighbourhoodProcessing import set_up_cube
 
 
+def _mean_points(points):
+    """Create an array of the mean of adjacent points in original array"""
+    mean_points = []
+    for i in range(0, len(points) - 1):
+        point = (points[i] + points[i + 1]) / 2
+        mean_points.append(point)
+
+    return np.array(mean_points, dtype=np.float32)
```

Comment:
This could shortened as below, similar to [recursive_filter.py#L325](https://github.com/metoppv/improver/blob/master/improver/nbhood/recursive_filter.py#L325)
```suggestion
    return np.array((points[:-1] + points[1:]) / 2, dtype=np.float32)
```

Reply:
That's a nice way of doing it, thank you.

Updated in commit b26bc9101b0cef970e4c11c1cea1946d3af29f67.